src/MCPServer/lib/unit.py

```diff
@@ -89,7 +89,7 @@ def setVariable(self, variable, variableValue, microServiceChainLink):
                         microServiceChainLink)
             for var in variables:
                 var.variablevalue = variableValue
-                var.microservicechainlink_id = microServiceChainLink
+                var.microservicechainlink = microServiceChainLink
```

Comment:
This is the stuff I mentioned about fields like `UnitVariable.microservicechainlink` becoming simple text fields instead of FKs, because the foreign tables related to workflow are being removed.